zh-Hans.json

```diff
@@ -1488,6 +1488,7 @@
 	"property.edgeLogic.shortCut": "键盘快捷键：",
 	"property.edgeLogic.showHelp": "显示光标处指令的文档",
 	"property.edgeLogic.softWrap": "文本软换行",
+	"property.enableIPv6Origin.alertMessage.usedIpv6OriginServer": "您已在源服务器中使用 IPv6 地址。您的资产必须启用使用 IPv6 访问源。",
```

Comment:
Please change it to:
您在源站服务器中配置了IPv6地址，此处必须开启“使用IPv6源站”。

Reply:
@Jianghai-Quantil 

Based on Burton's previous suggestion, I just updated "property.enableIPv6Origin.alertMessage.usedIpv6OriginServer" to "Since you have specified an IPv6 address for an origin server, you must set \"Enable IPv6 Origin\" to \"enable\"."

Can you please confirm if the following Chinese translation is accurate:
由于您已为源服务器指定了IPv6地址，因此必须将“启用IPv6源”设置为“启用”。